hazelcast/src/main/java/com/hazelcast/config/WanReplicationRef.java

```diff
@@ -15,10 +15,18 @@
  */
 
 package com.hazelcast.config;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.nio.serialization.DataSerializable;
+
+import java.io.IOException;
+import java.io.Serializable;
+
 /**
  * Configuration for Wan target replication reference
  */
-public class WanReplicationRef {
+public class WanReplicationRef implements DataSerializable, Serializable {
     private String name;
```

Comment:
@mesutcelik , `CacheConfiguration` interface extends `CompleteConfiguration` interface (from JCache API) which is `Serializable`. So all of its non-transient fields needs to be `Serializable`. (like `CacheEvictionConfig` and `NearCacheConfig`)